bedrock/firefox/templates/firefox/features/index.html

```diff
@@ -93,6 +93,11 @@ <h3>{{ _('Balanced memory usage') }}</h3>
           <h3>{{ _('Browse faster') }}</h3>
         </a>
       </li>
+      <li class="features-list-item compare">
```

Comment:
This is introducing a new string, so we'll need towrap an L10n conditional tag around this block.

https://bedrock.readthedocs.io/en/latest/l10n.html#staging-copy-changes

Give the tag a unique name, such as `firefox_features_compare_032020`